requirements.txt

```diff
@@ -1,3 +1,5 @@
 accelerate>=1.4.0
 datasets>=3.0.0
-transformers>=4.56.1
\ No newline at end of file
+transformers>=4.56.1
+math-verify==0.5.2
+latex2sympy2_extended>=1.0.6
```

Comment:
```suggestion
```

Reply:
Done! Also for my understanding why do we even have a `requirements.txt`? Aren't all the deps already correctly listed in `pyproject.toml`?

Reply:
tbh, I'm not sure if we really need `requirements.txt`. It's probably legacy